beacon-chain/cache/tracked_committee_indices.go

```diff
@@ -0,0 +1,44 @@
+package cache
```

Comment:
The more I think about this. To reduce disparity and introduce another cache where it's not really needed, we don't need a separate cache to track committee indices. We should just reuse the committee cache
https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/cache/committee.go

You can get the seed of current epoch and next epoch pretty easy

Reply:
It is also updated with next epoch look ahead in mind:
https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/core/helpers/committee.go#L370

Reply:
To retrieve the committee you want, just do:
```go
seed, err := Seed(state, epoch, params.BeaconConfig().DomainBeaconAttester)
if err != nil {
	return err
}
committees, exists, err := committeeCache.CommitteeCache.GetByKey(string(seed[:]))
```
The seed retrieval is really fast as it's already cached

Reply:
I miss-understood the purpose of this since committee_indices in spec widely refers the validator indices in the committee. 

Can we rename this to committee_ids.go?